ramls/logEvent.json

```diff
@@ -8,14 +8,16 @@
       "type": "string",
       "description": "The system assigned unique ID of the instance record; UUID"
     },
-    "eventCode": {
-      "type": "string",
-      "description": "Event code: PASSWORD_RESET, PASSWORD_CREATE"
-    },
-    "timestamp": {
-      "type": "string",
-      "format": "date-time",
-      "description": "Event registration time"
+    "eventType": {
+      "enum": [
+        "PASSWORD_RESET",
+        "PASSWORD_CREATE",
+        "PASSWORD_CHANGE",
+        "SUCCESSFUL_LOGIN_ATTEMPT",
+        "FAILED_LOGIN_ATTEMPT",
+        "USER_BLOCK"
+      ],
+      "description": "Event type"
     },
     "tenant": {
```

Comment:
'tenant' property is probably unnecessary is this schema as storage is tenant-specific.